combined_robot_hw_tests/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(combined_robot_hw_tests)
```

Comment:
It's not strictly necessary, but the fact that we are exporting plugins to be used only in the tests (MyRobotHW1, MyRobotHW2...) might make the package interface look untidy, so I followed the example of `controller_manager_tests` that I assumed had been created for the same reason.